assets/src/components/AudioExamples.js

```diff
@@ -2,86 +2,104 @@ import React from "react"
 import { Typography, Box, Grid, Avatar } from "@material-ui/core"
 import wave from "../../static/wave-orca.png"
 
-const AudioExamples = props => {
-  return (
-    <Grid
-      className="audio-example-section"
-      component="section"
-      container
-      direction="column"
-      justify="space-evenly"
-      alignItems="flex-start"
-      item
-    >
+class AudioExamples extends React.Component {
```

Comment:
If possible, it'd also be great to start using function components.

Reply:
Further down I see that you're defining the two button press events -- what do you think about this?

```
  handleButtonPressed = (event_name, title) => {
    gtag('event', `${event_name} @ ${title} player, homepage`, {'event_category' : 'custom'})
  }
```

(and further during the event binds)

```
onPlay={() => handleButtonPressed("play_button_pressed", example.title)}
onPause={() => handleButtonPressed("pause_button_pressed", example.title)}
```

Reply:
I turned the AudioExamples component into a function component. I'm not sure how to turn the the GoogleAnalytics component into a function component because it uses componentWillUpdate().

Reply:
I removed the onPause() button press event from this component, after talking with some members of the UI/UX team who are interested in this PR.